app/forms/further_education_payments/providers/claims/verification/check_answers_form.rb

```diff
@@ -149,6 +149,10 @@ def save
             # this has been completed.
             if claim.eligibility.provider_verification_claimant_employment_check_declaration
               Policies::FurtherEducationPayments.alternative_idv_completed!(claim)
+            elsif claim.academic_year != AcademicYear.new("2024/2025")
```

Comment:
i think this guard is not needed? as it looks like you've already got a guard on `#perform` on the automated check. i think if we delegate the responsibility down each job/check can decide if it wants to process the claim or not, pushing things down the stack